content/en/docs/reference/glossary/resource-driver.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Resource Driver
+id: resource-driver
+date: 2023-10-16
+full_link: /docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers/
+short_description: >
+  Software extensions to let Pods access devices that need vendor-specific initialization or setup
```

Comment:
This doesn't sound quite right. You might use a resource driver just for the allocation aspect, without any vendor specific setup.

For example, using DRA to assign a tape robot to a Pod that is running backup software, and exposing a vendor neutral protocol for tape management (eg: iSCSI).